aip/auth/4110.md

```diff
@@ -124,38 +124,38 @@ digraph d_front_back {
       1. If set, use the provided API key _[END]_
    3. If neither of above is set, go to step (2)
 2. **Load credentials**
-   1. Check gcloud default credentials through its default path
-      1. If found go to step (2.2)
-      2. Otherwise go to step (3)
-   2. Check the provided credential type
-      1. If the credential is gcloud credentials, go to step (4)
-      2. If the credential is a service account key JSON, go to step (4)
-      3. If the credential is unknown type, return an error saying that _[END]_
+  1. Check gcloud default credentials through its default path
+    1. If found go to step (2.2)
+    1. Otherwise go to step (3)
```

Comment:
```suggestion
    2. Otherwise, go to step (3)
```

Wouldn't this be 2 instead?

Reply:
This is intended. The index will be increased automatically by markdown.